src/App.js

```diff
@@ -12,10 +13,11 @@ function App() {
         <Route path='/' element = {
           <>
           <Header />
-          <Story />
+          <LandingPage />
           </>
           } 
           />
+          <Route path='/roomID' element={<Story/>}/>
```

Comment:
nice job adding a new route here! We'll make this dynamic after we get the endpoint from BE!